Bookish/Controllers/BookController.cs

```diff
@@ -0,0 +1,88 @@
+using System.Diagnostics;
+using Microsoft.AspNetCore.Mvc;
+using Bookish.Models;
+using Microsoft.AspNetCore.Authorization.Infrastructure;
+
+namespace Bookish.Controllers;
+
+public class BookController : Controller
+{
+    public IActionResult Catalogue()
+    {
+        using var context = new LibraryContext();

```

Comment:
It might be nice if we have all database-interacting code in other classes, like we do with the EditBooks and SearchBooksModel classes

Reply:
Created a method in EditBooks to do this (GetBookList)